inlang/source-code/website/src/server/main.ts

```diff
@@ -0,0 +1,48 @@
+import express from "express"
+import compression from "compression"
+import { validateEnvVariables, privateEnv } from "@inlang/env-variables"
+import * as Sentry from "@sentry/node"
+import * as Tracing from "@sentry/tracing"
+import { isProduction } from "./env.js"
+import { router } from "./router.js"
+// --------------- SETUP -----------------
+
+const { error: errors } = validateEnvVariables({ forProduction: isProduction })
+
+if (errors) {
+	throw Error(
+		"Production env variables are missing:\n\n" +
+			errors.map((e) => `${e.key}: ${e.errorMessage}`).join("\n")
+	)
+}
+
+const app = express()
+// compress responses with gzip
+app.use(compression())
+
+// setup sentry error tracking
+// must happen before the request handlers
+if (isProduction) {
+	Sentry.init({
+		dsn: privateEnv.SERVER_SENTRY_DSN,
+		integrations: [
+			// enable HTTP calls tracing
+			new Sentry.Integrations.Http({ tracing: true }),
+			// enable Express.js middleware tracing
+			new Tracing.Integrations.Express({ app }),
+		],
+		tracesSampleRate: 0.1,
+	})
+
+	// RequestHandler creates a separate execution context using domains, so that every
+	// transaction/span/breadcrumb is attached to its own Hub instance
+	app.use(Sentry.Handlers.requestHandler())
+	// TracingHandler creates a trace for every incoming request
+	app.use(Sentry.Handlers.tracingHandler())
+}
+
+app.set("base", "/")
```

Comment:
what is base for?

Reply:
It is the same as in the vite dev server. We need these to define where the files in the public folder are located.